tools/cvm-image-rewriter/cloud-init/user-data.basic

```diff
@@ -14,7 +14,8 @@ power_state:
   condition: True
 
 runcmd:
-  - sed -i 's/PasswordAuthentication no/PasswordAuthentication yes/g' /etc/ssh/sshd_config
+  - sed -i 's/#PasswordAuthentication yes/PasswordAuthentication yes/g' /etc/ssh/sshd_config
+  - sed -i 's/KbdInteractiveAuthentication no/KbdInteractiveAuthentication yes/g' /etc/ssh/sshd_config
```

Comment:
What's the reason of making this change?

Reply:
This is to address the issue of failed SSH login.